### ISSUE TITLE
JSPM@0.16.6 bundle command

### ISSUE THREAD

**Issue by stoiet** [jspm/jspm-cli#1143]
Hy!

I upgraded to version 0.16.6, and this command does not work now:
```
jspm bundle  [source/javascript/**/*.js] client/dist/bundle-application.js --inject --no-mangle --inline-source-maps
```

After execution the config.js file:
```json
"dist/bundle-application.js": [
  "[object Object]"
]
```

Is it a bug or did i miss something?

**Comment by stoiet** [comment]
With jspm@0.16.2 the bundling works well. Between 0.16.2 and 0.16.6 there was systemjs/builder version upgrading, i think this issue belongs to the systemjs/builder.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/builder/commit/174b977a640f055681330abdfd8b564dfb57e78d.

**Comment by stoiet** [comment]
@guybedford Thanks!

**Comment by guybedford** [comment]
Released in 0.16.7.
